sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -922,33 +923,79 @@ def dropna(self, **kwargs):
   to_string = frame_base.wont_implement_method(
       pd.Series, 'to_string', reason="non-deferred-result")
 
-  def aggregate(self, func, axis=0, *args, **kwargs):
+  @frame_base.args_to_kwargs(pd.Series)
+  @frame_base.populate_defaults(pd.Series)
+  def aggregate(self, func, axis, *args, **kwargs):
+    if kwargs.get('skipna', False):
+      # Eagerly generate a proxy to make sure skipna is a valid argument
+      # for this aggregation method
+      _ = self._expr.proxy().aggregate(func, axis, *args, **kwargs)
```

Comment:
If I understand this correctly, this forwards the arguments to an empty DataFrame to validate the arguments with a given aggregation function (e.g. sum, proc, etc). Since the code here seems to be duplicated in other functions, does it make sense to automate or at least break it out into a helper function?